spec/integration/guest_view_spec.rb

```diff
@@ -218,6 +218,19 @@
       expect(page).to have_content("$5.00")
       expect(page).to have_content("John Bob Smith")
     end
+
+    xit "doesn't show past events" do
```

Comment:
why's this test skipped? the diff is a little weird, it makes it look like its a brand new commented out test.